portal-frontend/src/app/features/edit-submission/proposal/pfrs-proposal/pfrs-proposal.component.html

```diff
@@ -0,0 +1,471 @@
+      </app-info-banner>
+      <app-file-drag-drop
+        id="reclamation-plan"
+        [uploadedFiles]="reclamationPlan"
+        (uploadFiles)="attachFile($event, DOCUMENT.RECLAMATION_PLAN)"
+        (deleteFile)="deleteFile($event)"
+        (openFile)="openFile($event)"
+        [showErrors]="showErrors"
+        [isRequired]="true"
+        [allowMultiple]="true"
+      ></app-file-drag-drop>
+    </div>
+
+    <div class="full-row">
+      <label for="isExtractionOrMining">Is your proposal for aggregate extraction or placer mining?</label>
+      <div>
+        <mat-button-toggle-group
+          class="input"
+          (valueChange)="onChangeMiningExtraction($event)"
+          id="isExtractionOrMining"
+          formControlName="isExtractionOrMining"
+        >
+          <mat-button-toggle
+            class="toggle-button"
+            [ngClass]="{
+              'error-outline':
+                isExtractionOrMining.invalid && (isExtractionOrMining.dirty || isExtractionOrMining.touched)
+            }"
+            value="true"
+            >Yes
+          </mat-button-toggle>
+          <mat-button-toggle
+            class="toggle-button"
+            [ngClass]="{
+              'error-outline':
+                isExtractionOrMining.invalid && (isExtractionOrMining.dirty || isExtractionOrMining.touched)
+            }"
+            value="false"
+            >No
+          </mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+    </div>
+
+    <div class="full-row">
+      <label for="isExtractionOrMining">
+        Have you submitted a Notice of Work to the Ministry of Energy, Mines and Low Carbon Innovation (EMLI)?
+      </label>
+      <div class="subtext">If yes, please attach the Notice of Work in the section below.</div>
+      <div>
+        <mat-button-toggle-group
+          class="input"
+          (valueChange)="onChangeNoticeOfWork($event)"
+          id="hasSubmittedNotice"
+          formControlName="hasSubmittedNotice"
+        >
+          <mat-button-toggle
+            class="toggle-button"
+            [ngClass]="{
+              'error-outline': hasSubmittedNotice.invalid && (hasSubmittedNotice.dirty || hasSubmittedNotice.touched)
+            }"
+            value="true"
+            >Yes
+          </mat-button-toggle>
+          <mat-button-toggle
+            class="toggle-button"
+            [ngClass]="{
+              'error-outline': hasSubmittedNotice.invalid && (hasSubmittedNotice.dirty || hasSubmittedNotice.touched)
+            }"
+            value="false"
+            >No
+          </mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+    </div>
+
+    <div class="full-row">
+      <label class="subheading2" for="noticeOfWork">Notice of Work</label>
+      <app-info-banner>
+        Please refer to <a>TODO: Removal of Soil</a> on the ALC website for more information
```

Comment:
Do we have a ticket to fill up this links?

Reply:
I believe so, this is pending on the ALC launching their new website though which is TBD